infrastructure/dogfood/terraform/aws-tf-module/main.tf

```diff
@@ -259,3 +259,8 @@ module "notify_slack" {
   slack_channel     = "#help-p1"
   slack_username    = "monitoring"
 }
+
+module "ses" {
+  source = "github.com/fleetdm/fleet//terraform/addons/ses?ref=edwardsb-ses-mailer"
```

Comment:
Should switch this to `main` before this merges and this branch goes away, breaking dogfood deploys.

Reply:
just sanity checking if we don't need to do this anymore?